modules/react/button/lib/DeleteButton.tsx

```diff
@@ -1,31 +1,20 @@
 import * as React from 'react';
-import {ButtonColors, ButtonSize, ButtonOrAnchorComponent} from './types';
+import {ButtonColors} from './types';
 import {ButtonContainer, ButtonLabel} from './parts';
 import {
   GrowthBehavior,
   useTheme,
   Themeable,
   EmotionCanvasTheme,
+  createComponent,
 } from '@workday/canvas-kit-react/common';
 
-export interface DeleteButtonProps
-  extends React.ButtonHTMLAttributes<HTMLButtonElement>,
-    Themeable,
-    GrowthBehavior {
+export interface DeleteButtonProps extends Themeable, GrowthBehavior {
   /**
    * The size of the Button.
-   * @default 'medium'
```

Comment:
Why remove the default?

Reply:
Default is handled by prop tables. I was tempted to get rid of the documentation of props in READMEs, but I wanted to do that with the migration to MDX and I didn't want to do that as part of this PR. If you feel strongly, we can revert the changes to `@default` in JSDoc

Reply:
Ugh. I'll reverse all of these. It turns out React docgen doesn't understand this very well.